src/main/java/command/Command.java

```diff
@@ -0,0 +1,52 @@
+package command;
+
+import java.util.Random;
+
+import task.Task;
+import task.TaskList;
+
+public abstract class Command {
+    private final Random RNG = new Random();
+    private int i;
```

Comment:
Perhaps you could rename `i` to a more descriptive name since it has large scope

Reply:
That makes sense, I'll rename it to `taskIndex` since that's what it refers to.